docs/_documentations/troubleshooting.md

```diff
@@ -783,22 +783,22 @@ Issue link: https://github.com/eclipse/codewind/issues/1682
 Solution: https://github.com/eclipse/codewind/issues/1682#issuecomment-572649328
 Info added in 0.11.0.
 -->
-Codewind will not work if it cannot find the `docker` executable in the [`PATH`](https://en.wikipedia.org/wiki/PATH_(variable)). You will see an error message like the following:
+Codewind will not work if it cannot find the `docker` executable in any of the paths specified in the `PATH` environment variable. You will see an error message like the following:
```

Comment:
Present tense:
- "Codewind does not work if it cannot find..."
- "You see an error message..."